core/test-runtime/src/lib.rs

```diff
@@ -248,6 +248,8 @@ cfg_if! {
 				fn use_trie() -> u64;
 				fn benchmark_indirect_call() -> u64;
 				fn benchmark_direct_call() -> u64;
+				fn returns_mutable_static() -> u64;
+				fn allocates_stack_alot(trap: bool);
```

Comment:
```suggestion
				fn allocates_huge_stack_array(trap: bool);
```

Or something similar.